### ISSUE TITLE
Cannot write files (Boxer 3.4.8 MacOS Mojave 14.6)

### ISSUE THREAD

**Issue by chaals** [boxer-project/boxer-sunrise#58]
Using the binary build on MacOS Mojave, it seems that nothing is ever written when I try to save/export/...

I tried running from the command line and `sudo` from the command line.

Expected behaviour: save/export options write a file
Observed behaviour: they open a save dialog and look like they will write something. But don't.

**Comment by sgithens** [comment]
Hi.  Yeah, actually, this was also reported by another person.  I'm *going* to try and fix it, I think it may have been something from the lispworks 8.0 update. 

It's uncertain how long we'll be able to go back and support unsupported releases of macOS.  At the moment, I do want to keep Mojave working, I believe the sort of mostly official answer is we'll make sure to support macOS n and n - 1 versions, but hopefully all currently supported versions of macOS, at the moment back to Catalina. Again, there's no reason not to support things back to Mojave and maybe High Sierra, especially since there is nothing we use that requires more than that, but as time goes on they will certainly get less testing.

Long story short, I do hope to have loading files working again on Mojave in the next release.  Interestingly, loading from the "recent files..." menu works, so I believe it's just some minor issue with the open dialogue usage.
